### DISCUSSION TITLE
Xworkspaces module not working on i3

### DISCUSSION THREAD

**Discussion by ghost** [polybar/polybar#2808] [Q&A] [answered]
I want to change the icons numbers of the workspace. This is my config file - (https://0x0.st/op47.txt). It still didn't change the numbers of the workspace (1,2,3,4) to the desired icons :(. Any help would be appreciated.

**Comment by patrick96** [comment, marked as answer]
You need to use the `%icon%` token, the `%name%` token contains the name provided by i3.

Also, for the icon mapping, the thing before the semicolon needs to match the workspace name. So either you need to rename your workspaces to `code`, `office`, etc. or you need to use the correct names in `icon-*`.

**Comment by ghost** [comment]
>You need to use the %icon% token, the %name% token contains the name provided by i3.

>Also, for the icon mapping, the thing before the semicolon needs to match the workspace name. So either you need to rename your workspaces to code, office, etc. or you need to use the correct names in icon-*.

Oh ok but what do you mean by `you need to use the correct names in icon-*` ?

**Comment by ghost** [comment]
Nevermind got it
